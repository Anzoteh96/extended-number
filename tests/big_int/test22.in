8917700074089121821086357752945065877704649586254873981576530770029
681909818249593256472933537397987524612769607155607125068572474905
