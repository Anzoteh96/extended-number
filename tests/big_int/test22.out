Enter two numbers n and m:
n is 8917700074089121821086357752945065877704649586254873981576530770029
m is 681909818249593256472933537397987524612769607155607125068572474905
n + m is 9599609892338715077559291290343053402317419193410481106645103244934
n - m is 8235790255839528564613424215547078353091879979099266856507958295124
n * m is 6081067236726497378537651368592516139198084513254970131510889898479266564452550926601825232763865458835785608236842945189773928622245
n / m is 13
n % m is 52872436844409486938221766771228057738644693231981355685088596264
